playbooks/roles/azure-security-group/tasks/main.yml

```diff
@@ -45,6 +45,15 @@
         access: Allow
         direction: Inbound
         priority: 108
+      # Let's Encrypt
+      # ---
+      - name: "AllowLetsEncrypt"
```

Comment:
I think I would prefer we call the rule "AllowHTTP" and then comment that it's for Let's Encrypt. Does that make sense to you?